lighthouse-core/test/fraggle-rock/api-test-pptr.js

```diff
@@ -194,7 +194,7 @@ describe('Fraggle Rock API', () => {
       const {lhr} = result;
       const {auditResults, failedAudits, erroredAudits} = getAuditsBreakdown(lhr);
       // TODO(FR-COMPAT): This assertion can be removed when full compatibility is reached.
-      expect(auditResults.length).toMatchInlineSnapshot(`152`);
```

Comment:
This is `TapTargets` right?

Reply:
yep, the inheriting supportedModes array is `[]`